renderdoc/driver/vulkan/vk_info.cpp

```diff
@@ -792,6 +792,10 @@ void VulkanCreationInfo::Pipeline::Init(VulkanResourceManager *resourceMan,
                                         const VkGraphicsPipelineCreateInfo *pCreateInfo)
 {
   flags = pCreateInfo->flags;
+  auto *pPipelineFlags2 =
+      FindNextStruct(pCreateInfo, VK_STRUCTURE_TYPE_PIPELINE_CREATE_FLAGS_2_CREATE_INFO_KHR);
```

Comment:
Please don't use `auto`, instead specify the type and do the cast at the declaration instead of below when accessing it.